### DISCUSSION TITLE
Help with autocomplete-moose

### DISCUSSION THREAD

**Discussion by marinasessim** [idaholab/moose#17778] [Q&A General] [answered]
Hello,

This is related to the autocomplete-moose atom package. First, I do not think there is an issue with the package, it just does not work for my application. It works properly inside the moose directory, for instance, within the phase-field module. But it does not work for input files in my moose-based app.

I am wondering if there is anything I could have done to a makefile or something that would prevent autocomplete-moose from working. It does not return any errors, it just does not autocomplete. The last version that worked for me was v0.23.3. I got away using that one until now, but I just performed some updates and unfortunately, that version no longer works (neither does the newest one). I am just trying to get some advice from the community because I do not know what else to try. 

Current macOS BigSur 11.3.1; CommandLine Tools for Xcode 12.4; Atom v1.56.0.

My application runs normally otherwise. I have tried generating a syntax.txt, pointing the opt directory, everything that I could think of. 

I appreciate any help! Thanks!

**Comment by aeslaughter** [comment]
@dschwen Can you help?

**Comment by hugary1995** [comment]
Before @dschwen gets to this, can you try `yourApp-opt --json` and tell us the output? Meanwhile, you should create an issue at https://github.com/dschwen/autocomplete-moose.

**Reply by marinasessim** [reply]
Hello, thanks for your attention. I am attaching the --json output.

[syntax.txt](https://github.com/idaholab/moose/files/6479466/syntax.txt)

**Comment by marinasessim** [comment, marked as answer]
Hello, just sharing the resolution to this problem. The input file was in the old format. The file was updated to the moose hit format using the moose-hit-format package, and now autocomplete-moose works well. Thanks!!
